src/systemctl/systemctl-logind.c

```diff
@@ -45,18 +47,21 @@ int logind_reboot(enum action a) {
 #if ENABLE_LOGIND
         static const struct {
                 const char *method;
+                const char *method_with_flags;
                 const char *description;
         } actions[_ACTION_MAX] = {
-                [ACTION_POWEROFF]               = { "PowerOff",             "power off system"                },
-                [ACTION_REBOOT]                 = { "Reboot",               "reboot system"                   },
-                [ACTION_HALT]                   = { "Halt",                 "halt system"                     },
-                [ACTION_SUSPEND]                = { "Suspend",              "suspend system"                  },
-                [ACTION_HIBERNATE]              = { "Hibernate",            "hibernate system"                },
-                [ACTION_HYBRID_SLEEP]           = { "HybridSleep",          "put system into hybrid sleep"    },
-                [ACTION_SUSPEND_THEN_HIBERNATE] = { "SuspendThenHibernate", "suspend system, hibernate later" },
+                [ACTION_POWEROFF]               = { "PowerOff",             "PowerOffWithFlags",             "power off system"                },
+                [ACTION_REBOOT]                 = { "Reboot",               "RebootWithFlags",               "reboot system"                   },
+                [ACTION_HALT]                   = { "Halt",                 "HaltWithFlags",                 "halt system"                     },
+                [ACTION_SUSPEND]                = { "Suspend",              "SuspendWithFlags",              "suspend system"                  },
+                [ACTION_HIBERNATE]              = { "Hibernate",            "HibernateWithFlags",            "hibernate system"                },
+                [ACTION_HYBRID_SLEEP]           = { "HybridSleep",          "HybridSleepWithFlags",          "put system into hybrid sleep"    },
+                [ACTION_SUSPEND_THEN_HIBERNATE] = { "SuspendThenHibernate", "SuspendThenHibernateWithFlags", "suspend system, hibernate later" },
```

Comment:
i must say, I'd probably have left the table as is, and concatenated the `WithFlags` to it with a quick strjoina() when needed.